src/aap_eda/core/utils/k8s_service_name.py

```diff
@@ -0,0 +1,94 @@
+#  Copyright 2024 Red Hat, Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import re
+import secrets
+
+SVC_PREFIX = "eda"
+PEAKS = [
+    "everest",
+    "k2",
+    "kanchenjunga",
+    "lhotse",
+    "makalu",
+    "dhaulagiri",
+    "manaslu",
+    "annapurna",
+    "gasherbrum",
+    "himalchuli",
+    "distaghil",
+    "nuptse",
+    "masherbrum",
+    "rakaposhi",
+    "kamet",
+    "kumbhakarna",
+    "molamenqing",
+    "chogolisa",
+    "shispare",
+    "trivor",
+    "changtse",
+    "noshaq",
+    "malubiting",
+    "gangapurna",
+    "k12",
+    "yangra",
+    "kabru",
+    "ultar",
+    "rimo",
+    "saraghar",
+    "talung",
+    "jomolhari",
+    "chamlang",
+    "chongtar",
+    "k6",
+    "kangpenqing",
+    "mana",
+    "diran",
+    "karjiang",
+]
+
+DEPRESSIONS = [
+    "calypso",
+    "challenger",
+    "dordrecht",
+    "emden",
+    "litke",
+    "horizon",
+    "marianas",
+    "sirena",
+    "milwaukee",
+    "molloy",
+    "atacama",
+]
+
+
+def create_k8s_service_name(activation: str) -> str:
```

Comment:
@hsong-rh We don't need this since we are using service name from activation name.